src/components/Main/Main.tsx

```diff
@@ -58,6 +63,16 @@ const Main = () => {
       })
   }, []);
 
+  useEffect(() => {
+    for (let letter = 'A'.charCodeAt(0); letter <= 'J'.charCodeAt(0); letter++) {
+      for (let number = 1; number <= 10; number++) {
+        const key = String.fromCharCode(letter) + number;
+        (lastUpdate as Record<string, string>)[key] = '10/10/2000 10:10';
+      }
+    }
+    // eslint-disable-next-line
+  }, []);
```

Comment:
I'm not exactly sure what this is doing... why are we setting a hardcoded datetime string to everything?